### DISCUSSION TITLE
Error 500 and FatalThrowableError in PagesDriver.php

### DISCUSSION THREAD

**Discussion by pedro-club** [statamic/cms#8115] [Troubleshooting]
Hi everyone,
Since yesterday, I get a "Whoops, looks like something went wrong." message for every page in my [website](https://www.centrenationaldugraphisme.fr/).
It returns systematically a ```500``` error code.
I'm running in _Statamic v2_, latest version.

It started after updating a post in a collection.

After turning the debug mode to true, I get this message : 
```
FatalThrowableError in PagesDriver.php 
line 86: Call to a member function in() on null
```
which is this line of code : 
```php
$uri = $parent->in($locale)->uri(); 
```
Every error messages seems to come from Stache folder files. I don't know if that info can help...
All debug is currently visible [here](https://www.centrenationaldugraphisme.fr/).

I replaced the website by a backup version (no error at the time), but I still get the error.
I can't figure out where does this error come from...

Any suggestions ?
Thanks !

**Comment by jasonvarga** [comment]
Clear your cache and try again. `php please clear:cache`

**Reply by pedro-club** [reply]
I get this error :
```
[Symfony\Component\Debug\Exception\FatalThrowableError]  
Call to a member function in() on null 
```

**Reply by jasonvarga** [reply]
Please open a support request and we'll help you there. If you can provide us with a copy of the site that'll be even more helpful. This repo is for Statamic 3+.

https://statamic.com/support
